src/app/app.component.html

```diff
@@ -0,0 +1,40 @@
+<nav>
```

Comment:
@hitpopdimestop Okay, it was just the old way I think, as I have seen that before in Angular 1 projects.

I found your naming convention in Angular style guide https://angular.io/styleguide#!#naming

So let us rename the other file